data/migrations/deb/1_2_233_to_1_2_234.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3.5
+import os
+
+from stp_core.common.log import getlogger
+from indy_common.config_util import getConfig
+
+import indy_node.general_config.indy_config as indy_config
+
+
+def migrate():
+    config = getConfig()
+    logger = getlogger()
+
+    general_config_path = os.path.join(config.GENERAL_CONFIG_DIR, config.GENERAL_CONFIG_FILE)
+
+    if os.path.exists(general_config_path):
```

Comment:
That's not a big problem for migration script and this method, but in general I prefer not having too many embedded conditions, and rather use 
```
if not os.path.exists(general_config_path):
    return
```

Reply:
Ok